PYTHON/manifest/manifest.py

```diff
@@ -79,7 +95,9 @@ def func(name: str, input_type: str):
                 )
 
             params[name] = {
-                "type": get_full_type_name(param_type),
+                "type": "node_reference"
```

Comment:
We should find a better way to encode the fact that a parameter is a node reference instead of hardcoding nodes that it should work for.

Reply:
Introduced ParameterTypes class to handle this.